Utilities/GetPSGalleryItemsForAuthor.ps1

```diff
@@ -0,0 +1,100 @@
+﻿
+<#PSScriptInfo
+
+.VERSION 0.1.0
```

Comment:
This needs to 1.0.0 as this is not a prerelease version, right?

Reply:
Actually, this version IS prerelease, while we work out the bugs you catch